setup.py

```diff
@@ -91,13 +91,28 @@ def generate_proto(source):
         'australis': ['protobuf>=3.13,<4'],
 
         # CUDA installations require adding jax releases URL; e.g.
-        # Cuda installation defaulting to a CUDA and Cudnn version defined above.
+        # Batteries-included CUDA installation defaulting to a CUDA and Cudnn
+        # version defined above.
         # $ pip install jax[cuda] -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
-        'cuda': [f"jaxlib=={_current_jaxlib_version}+cuda{_default_cuda_version}.cudnn{_default_cudnn_version}"],
+        'cuda': [
+          f"jaxlib=={_current_jaxlib_version}+cuda{_default_cuda_version}.cudnn{_default_cudnn_version}",
+          "nvidia-cublas-cu11",
+          "nvidia-cuda-nvcc-cu11",
+          "nvidia-cuda-runtime-cu11",
+          "nvidia-cudnn-cu11",
+          "nvidia-cufft-cu11",
+          "nvidia-cusolver-cu11",
+          "nvidia-cusparse-cu11",
+        ],
+
+        # Target that does not depend on the CUDA pip wheels, for those who want
+        # to use a preinstalled CUDA.
+        'cuda_using_existing_cuda_installation': [
+          f"jaxlib=={_current_jaxlib_version}+cuda{_default_cuda_version}.cudnn{_default_cudnn_version}",
+        ],
 
         # CUDA installations require adding jax releases URL; e.g.
         # $ pip install jax[cuda11_cudnn82] -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
-        # $ pip install jax[cuda11_cudnn805] -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
```

Comment:
Why remove this comment if its not covered by cuda installation?

Reply:
I guess it's unrelated to this change, but we can/should drop that wheel now from our build. The reason is that we can simply say "if your CUDA installation is that old, use the pip version instead".

Reverted for now, though.